docs/source/markdown/podman-system-service.1.md

```diff
@@ -41,5 +41,5 @@ podman system service --timeout 5000
 podman(1), podman-system-service(1), podman-system-connection(1)
 
 ## HISTORY
-January 2020, Originally compiled by Brent Baude<bbaude@redhat.com>
-November 2020, Updated by Jhon Honce <jhonce at redhat.com>
+January 2020, Originally compiled by Brent Baude `<bbaude@redhat.com>`
+November 2020, Updated by Jhon Honce `<jhonce@redhat.com>`
```

Comment:
@jwhonce does this work for you?

Reply:
Thanks for the comment, @TomSweeneyRedHat . `grep -R honce docs` shows that Jhon has an established convention for his email address in md files, so I've updated `podman-system-service.1.md` accordingly and re-pushed.